front/app/containers/Admin/projects/project/phaseSetup/components/PhaseParticipationConfig/components/inputs/NativeSurveyInputs.tsx

```diff
@@ -94,39 +86,11 @@ const NativeSurveyInputs = ({
           </Box>
         }
       />
-      {userFieldsInSurveysEnabled && (
-        <SectionField>
-          <SubSectionTitle style={{ marginBottom: '0px' }}>
-            <FormattedMessage {...messages.userFieldsInSurveyTitle} />
-          </SubSectionTitle>
-          <Toggle
-            checked={user_fields_in_form || false}
-            onChange={() => {
-              handleUserFieldsInFormOnChange(!user_fields_in_form);
-            }}
-            label={
-              <Box ml="8px" id="e2e-user-fields-in-form-toggle">
-                <Box display="flex">
-                  <Text
-                    color="primary"
-                    mb="0px"
-                    fontSize="m"
-                    fontWeight="semi-bold"
-                  >
-                    <FormattedMessage {...messages.userFieldsInSurveyToggle} />
-                  </Text>
-                </Box>
 
-                <Text color="coolGrey600" mt="0px" fontSize="m">
-                  <FormattedMessage
-                    {...messages.userFieldsInSurveyDescription}
-                  />
-                </Text>
-              </Box>
-            }
-          />
-        </SectionField>
-      )}
+      <UserFieldsInSurveyToggle
+        user_fields_in_form={user_fields_in_form}
```

Comment:
Does this mean that this will be available for regular Native Surveys as well then once released?

Reply:
Yes, once released and enabled